Form/Type/ChangePasswordFormType.php

```diff
@@ -32,7 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'form.current_password',
             'translation_domain' => 'FOSUserBundle',
             'mapped' => false,
-            'constraints' => $constraint,
+            'constraints' => array(new NotBlank(), $constraint),
```

Comment:
please use a multiline array, as used in master. It will make it easier to handle merging branches by making the code more similar